### PR TITLE
CRUD operations

### PR DESCRIPTION
**Pull request by svshriyansh**

### CODE REVIEW

File: routes/api/users.js [hunk 1]
```diff
@@ -34,4 +35,70 @@ router.post("/", async (req, res) => {
     }
 });
 
+router.get("/:email", async (req, res) => {
```
Comment (svsaurabh) [inline routes/api/users.js:41]: Please add method signatures

File: routes/api/users.js [hunk 2]
```diff
@@ -34,4 +35,70 @@ router.post("/", async (req, res) => {
...
+                .status(400)
+                .json({ errors: [{ mesage: "User does not exist" }] });
+        }
+        res.status(200).send(user);
+    } catch (err) {
+        res.status(500).json({ message: "Internal Server error" });
+    }
+});
+
+router.get("/", async (req, res) => {
```
Comment (svsaurabh) [inline routes/api/users.js:58]: Please add method signatures

File: routes/api/users.js [hunk 3]
```diff
@@ -34,4 +35,70 @@ router.post("/", async (req, res) => {
...
+    try {
+        const user = await User.find();
+        res.json(user);
+    } catch (err) {
+        res.status(500).send("Internal Server Error");
+    }
+});
+
+router.put("/:email", async (req, res) => {
```
Comment (svsaurabh) [inline routes/api/users.js:70]: Please add method signatures

File: routes/api/users.js [hunk 4]
```diff
@@ -34,4 +35,70 @@ router.post("/", async (req, res) => {
...
+            message: "User has been successfully updated",
+            data: found,
+        });
+    } catch (err) {
+        console.log(err.message);
+        res.status(500).send("Internal Server error");
+    }
+});
+
+router.delete("/:email", async (req, res) => {
```
Comment (svsaurabh) [inline routes/api/users.js:96]: Please add method signatures

File: routes/api/users.js (hunk 1 above)
Comment (svshriyansh) [reply routes/api/users.js:41]: Resolved

File: routes/api/users.js (hunk 2 above)
Comment (svshriyansh) [reply routes/api/users.js:58]: Resolved

File: routes/api/users.js (hunk 3 above)
Comment (svshriyansh) [reply routes/api/users.js:70]: Resolved

File: routes/api/users.js (hunk 4 above)
Comment (svshriyansh) [reply routes/api/users.js:96]: Resolved

**Review by svsaurabh — APPROVED** [review]

**Pull request merged**